Pages/LanguagePage.cs

```diff
@@ -0,0 +1,124 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project_Mars.Pages
+{
+    public class LanguagePage
+    {
+        public void CreateLanguageRecord(IWebDriver driver)
+        {
+            IWebElement addNewButton = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/thead/tr/th[3]/div"));
```

Comment:
Please do not pass the driver In action methods (methods which in pages doing things like create, delete etc ) instead get the pages to extend to common driver where you will create a common driver class which will initialise the web driver 

public class CommonDriver
    {
        public static IWebDriver driver;

        public void initialise()
        {
            // create driver
            driver = new ChromeDriver();

        }

    }

Reply:
and then in the hooks  create a common driver instance which calls the initialise method